finality-aleph/src/network/split.rs

```diff
@@ -230,12 +247,14 @@ fn split_receiver<
             translated_receiver: left_receiver,
             left_sender: left_sender.clone(),
             right_sender: right_sender.clone(),
+            name: left_name.to_string(),
```

Comment:
Hmm, couldn't you just pass the `&'static str` in here and not use `String` at all?

Reply:
changed